apps/settings/style/settings_large.css

```diff
@@ -65,10 +65,15 @@
   }
 
   section[role="region"]#root header:after {
-    background: url("images/ui/tablet_separator.png") no-repeat right / 0.2rem transparent;
+    background-image: url("images/ui/tablet_separator.png");
+    background-repeat: no-repeat;
+    background-size: 0.2rem;
+    background-color: transparent;
     height: 95%;
     top: 0;
   }
+  html[dir="ltr"] section[role="region"]#root header:after { background-position: right; }
+  html[dir="rtl"] section[role="region"]#root header:after { background-position: left; }
```

Comment:
I’m not aware of any work or spec to support `start/end` in `background-position`. If you have a reference, I’ll be happy to track it.